src/app/components/splash-screen/splash-screen.component.scss

```diff
@@ -0,0 +1,50 @@
+@import "src/styles/_variables.scss";
+@import "src/styles/_colors.scss";
+@import "src/styles/_mixins.scss";
+
+.app-splash-screen {
```

Comment:
Please follow https://github.com/sza-wtag/ToDoApp/pull/1/files#r1228083719

Reply:
used BEM in styling vaia.